### DISCUSSION TITLE
WebDavService is OK ？

### DISCUSSION THREAD

**Discussion by liqijian101** [luc-github/ESP3D-TFT#49] [Q&A] [answered]
Can I use WebDav now? Why can't I see the settings for WebDav on the webpage.
It cannot be connected through webdav either

**Comment by luc-github** [comment]
<s>Did you enabled the service in Configuration.h ?</s>
Sorry wrong repo

**Reply by luc-github** [reply, marked as answer]
Sorry previous answer was too fast - I am currently on ESP3D code orz...

Yes WebDav is working - on ESP3D-TFT unlike ESP3DD, it is based on http server, so there is currently no parameter because it use same port as webserver 
I may add ON/OFF like for WebSocket that actually also use http server 
the way to connect is to access the webdav handler `http://<your ip>/webdav`
![webdavconnection](https://github.com/luc-github/ESP3D-TFT/assets/8822552/66575bfb-3960-4a8f-864f-6b124572bb0c)

I will also write a page for it on esp3d.io - but should not be before next week because I am on big rewrite on esp3d 3.0 right now

**Reply by liqijian101** [reply]
I enable webdav on CMakeLists。
when i imput http://192.168.123.100/webdav in webui,  
Web page display“This is not a file”. It can not work when i add  network location

**Reply by luc-github** [reply]
that is normal you need a webdav client, it does not work with browser
about windows I am not sure what is your issue as it work on my side as you can see, I have also tested with filezilla pro and an android app
in window be sure you type the http:// not just the ip

**Reply by liqijian101** [reply]
I use WINSCP software, and I can use it correctly by entering IP in ESP3D.
NOw  ，in ESPD-TFT, I enter http://ip/webdav It can be used normally now
thanks

**Comment by luc-github** [comment]
I have just added the Setting for webdav:
![image](https://github.com/luc-github/ESP3D-TFT/assets/8822552/745a1532-3f9d-44f3-ae50-e3aa35544e0f)
and in [ESP420]
![image](https://github.com/luc-github/ESP3D-TFT/assets/8822552/cc9a64e8-5b11-45ba-a4eb-8d558a5b1c9b)
